uui-components/src/overlays/Tooltip.tsx

```diff
@@ -129,6 +130,25 @@ export class Tooltip extends React.Component<TooltipProps, TooltipState> {
         </div>;
     }
 
+    private renderChildren = ({ ref, placement, style, arrowProps, isReferenceHidden }: PopperChildrenProps) => {
+        if (isReferenceHidden && this.state.isOpen) {
```

Comment:
Please add comment : 
// Yes, we know that it's hack and we can perform setState in render, but we don't have other way to do it in this case